frontend/src/lib/components/apps/components/display/table/AppAggridTable.svelte

```diff
@@ -303,6 +303,11 @@
 							$componentControl[id] = {
 								agGrid: { api: e.api, columnApi: e.columnApi },
 								setSelectedIndex: (index) => {
+									if(index === -1) {
+										e.api.deselectAll();
+										outputs?.selectedRow?.set({})
+										outputs?.selectedRowIndex.set(-1)
+									}
```

Comment:
the rest should be in an else

Reply:
I've updated the selecting of an index to be in an else